src/App.js

```diff
@@ -0,0 +1,22 @@
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Header from './components/Header';
+import Missions from './pages/Missions';
+import Profile from './pages/Profile';
+import Rockets from './pages/Rockets';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/" exact element={<Rockets />} />
+        <Route path="/missions" element={<Missions />} />
+        <Route path="/profile" element={<Profile />} />
+      </Routes>
```

Comment:
- [ ] I noticed some issues in your app, for example when I reserve rockets and join missions if I refresh the page the data disappear to the `My profile page`, or if I navigate through the pages I have the same results as the reserved rockets are unreserve or the join missions return to the initial state and other is not joined missions or reserved rockets the data is supposed to be preserve. Please kindly fix that issue. I record I short video to show you the problem 🙃 😉 

https://user-images.githubusercontent.com/74501523/193062912-6ba564a2-7667-4b4a-916f-e3a60df6a07a.mov

Reply:
Hi @kendoriddy,

The problem in your case is that when I refresh the page or if I was on the mission page, and I have already joined some missions and then I change the page go to the rocket page, when I return to the previous page (the mission page) all the missions I had already joined are left or when I refresh the page all the rockets or missions I had already reserved are not reserved anymore, normally when I navigate through the application and if I have already reserved some rockets and join some missions even if I refresh the page these missions and rockets are supposed to stay reserved and the missions join but also display in the table on the `My profile page` so I have to decide if I have to leave mission or I have to cancel a reservation I don't know if it's clear 🙃

Reply:
We can accept the case that if we refresh the page data can disappear because we didn't preserve data but if the user navigates through the three pages the data can't disappear, in your case when for example I'm on the rocket page and I reserved some rockets and decide to return to the rocket page because I forget to reserve one rocket all the rockets that I reserved are no longer reserved and vis versa for missions. I think the good practice is the user can navigate through the three pages without losing their data but if the user refreshes the page now he can lose their data.

Reply:
Thank you very much for the feedback. The issue is resolved now 👍